src/main/java/io/github/thebusybiscuit/slimefun4/utils/ChargeUtils.java

```diff
@@ -8,32 +8,40 @@
 
 import javax.annotation.Nonnull;
 
+import io.github.thebusybiscuit.slimefun4.core.attributes.Rechargeable;
+import org.apache.commons.lang.Validate;
 import org.bukkit.NamespacedKey;
 import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.persistence.PersistentDataContainer;
 import org.bukkit.persistence.PersistentDataType;
 
 import io.github.thebusybiscuit.cscorelib2.chat.ChatColors;
 import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
-import io.github.thebusybiscuit.slimefun4.utils.PatternUtils;
 import net.md_5.bungee.api.ChatColor;
 
 /**
  * This is just a simple helper class to provide static methods to the {@link Rechargeable}
  * interface.
- * 
+ *
  * @author TheBusyBiscuit
- * 
+ * @author WalshyDev
+ *
  * @see Rechargeable
  *
  */
-final class RechargeableHelper {
+public final class ChargeUtils {
 
     private static final String LORE_PREFIX = ChatColors.color("&8\u21E8 &e\u26A1 &7");
     private static final Pattern REGEX = Pattern.compile(ChatColors.color("(&c&o)?" + LORE_PREFIX) + "[0-9.]+ / [0-9.]+ J");
 
-    private RechargeableHelper() {}
+    private ChargeUtils() {}
+
+    public static void setCharge(@Nonnull ItemMeta meta, float charge, float capacity) {
+        Validate.notNull(meta, "Meta cannot be null!");
+        Validate.isTrue(charge >= 0, "Charge has to be equal to or greater than 0!");
+        Validate.isTrue(capacity >= 0, "Capacity has to be equal to or greater than 0!");
```

Comment:
```suggestion
        Validate.isTrue(capacity > 0, "Capacity has to be greater than 0!");
```
A capacity of zero would not make any sense.
Then it would not be able to hold any charge.

Reply:
*facepalm* I'm smart